configuration/messageproperties/messages_zl_ht.properties

```diff
@@ -64,4 +64,5 @@ emr.formValidation.messages.phoneField=Nimewo telefòn envalid. Fòma ki otorize
 registrationapp.patient.phone.question=Ki nimewo telefòn pasyan an (xxxxxxxx oswa xxxx-xxxx oswa +xxx xxxxxxxx)?
 zl.registration.patient.contactPerson.phonenumber.question=Ki nimewo telefòn kontak pasyan an (xxxxxxxx oswa xxxx-xxxx oswa +xxx xxxxxxxx)?
 
-pihcore.yesOtherReason=Wi, lòt rezon
\ No newline at end of file
+pihcore.yesOtherReason=Wi, lòt rezon
+zl.consultNote.delivery.label=Akouchman
```

Comment:
@louidorjp Added "Akouchman" to the messages_ht.properties file.  You don't need to do this here.